### ISSUE TITLE
How long does it take your time to train the model?

### ISSUE THREAD

**Issue by deepmeng** [qianguih/voxelnet#27]
I train the model in a single Titan X GPU and about a week passed while it's not over yet. 
Guys, how long you cost?

**Comment by sakshi404** [comment]
I just used the pre-trained model. but yes I found the inference time much longer as compared to the value mentioned in paper when I run model on validation set. I just checked for two samples and session.run took 190 seconds but timing profile shows only around 25 -30 seconds for all the layers of RPN network. I was wondering whether data transfer/pre-processing (VFE network) is taking huge time or there is some other reason. (I run on CPU, but still there is huge difference between time taken by RPN and inference time)

**Comment by merriaux** [comment]
On CPU for one HDL64e point cloud, I have 340ms for data preprocessing and 55.4s for inference (sess.run)

It seems to be impossible to change the research ragne  (__C.Y_MIN __C.Y_MAX...) in config.py after training, someone could confirm that  ?
If someone have ever train the network for all classes, it is possible to him to upload the weighs somewhere ?
best,

**Comment by nyg2017** [comment]
@sakshi404 hi. I wonder to know if you can  achieve better performance by fine-tone with the pre--train model?
